lang/main.json

```diff
@@ -226,32 +226,36 @@
         "allow": "Allow",
         "kickMessage": "Ouch! You have been kicked out of the meet!",
-        "popupError": "Your browser is blocking popup windows from this site. Please enable popups in your browser's security settings and try again.",
+        "popupErrorTitle": "Pop-up blocked",
+        "popupError": "Your browser is blocking pop-up windows from this site. Please enable popups in your browser's security settings and try again.",
         "passwordErrorTitle": "Password Error",
         "passwordError": "This conversation is currently protected by a password. Only the owner of the conference can set a password.",
         "passwordError2": "This conversation isn't currently protected by a password. Only the owner of the conference can set a password.",
         "connectError": "Oops! Something went wrong and we couldn't connect to the conference.",
         "connectErrorWithMsg": "Oops! Something went wrong and we couldn't connect to the conference: __msg__",
-        "incorrectPassword": "Password is incorrect",
+        "incorrectPassword": "Incorrect username or password",
         "connecting": "Connecting",
         "copy": "Copy",
+        "contactSupport": "Contact support",
         "error": "Error",
         "createPassword": "Create password",
         "detectext": "Error when trying to detect desktopsharing extension.",
-        "failtoinstall": "Failed to install desktop sharing extension",
         "failedpermissions": "Failed to obtain permissions to use the local microphone and/or camera.",
         "conferenceReloadTitle": "Unfortunately, something went wrong.",
         "conferenceReloadMsg": "We're trying to fix this. Reconnecting in __seconds__ sec...",
         "conferenceDisconnectTitle": "You have been disconnected.",
         "conferenceDisconnectMsg": "You may want to check your network connection. Reconnecting in __seconds__ sec...",
+        "dismiss": "Dismiss",
         "rejoinNow": "Rejoin now",
-        "maxUsersLimitReached": "The limit for maximum number of members in the conference has been reached. The conference is full. Please try again later!",
+        "maxUsersLimitReachedTitle": "Maximum members limit reached",
+        "maxUsersLimitReached": "The limit for maximum number of members has been reached. The conference is full. Please contact the meeting owner or try again later!",
         "lockTitle": "Lock failed",
         "lockMessage": "Failed to lock the conference.",
         "warning": "Warning",
-        "passwordNotSupported": "Room passwords are currently not supported.",
+        "passwordNotSupportedTitle": "Password not supported",
+        "passwordNotSupported": "Setting a meeting password is not supported.",
         "internalErrorTitle": "Internal error",
-        "internalError": "Oups! Something went wrong. The following error occurred: [setRemoteDescription]",
+        "internalError": "Oups! Something went wrong. The following error occurred: __error__",
```

Comment:
I think "oups" might be the french spelling of "oops."

Reply:
It is indeed, I'll fix 👍